src/reuse/comment.py

```diff
@@ -400,6 +421,16 @@ class FtlCommentStyle(CommentStyle):
     MULTI_LINE = MultiLineSegments("<#--", "", "-->")
 
 
+class GleamCommentStyle(CommentStyle):
```

Comment:
I think `CppCommentStyle` will serve the same purpose here, won't it?

Reply:
I am not sure it would work, as Gleam does not have multiline as CPP
```
-- copy-pasted CppCommentStyle snippet
MULTI_LINE = MultiLineSegments("/*", "*", "*/")
```
and the CppCommentStyle also uses some php SHEBANG which I do not think work on Gleam
```
-- copy-pasted CppCommentStyle snippet
SHEBANGS = [
        "#!",  #  V-Lang
        "<?php",  # PHP
    ]
```

Reply:
If I'm not completely off, using `CppCommentStyle` would only cause a problem if there was a chance for something that looks like a multiline comment in Gleam, but isn't one. The same should also be true for the shebang. @carmenbianca can you confirm?

Reply:
To be honest, I do not think it is an issue in Gleam, but I have not written much Gleam myself.
I mention this as to separate concerns :) 
Please let me know what the preferred way would be :)

Reply:
The shebangs should not create any problems (I think it's unlikely you'll find `<?php` in Gleam code) as REUSE will never add them, simply change the position of the added header in case it encounters as shebang in the first line of the file.

The multiline comments would create an issue however, if REUSE was forced to create a multiline header. So I think `CppSingleCommentStyle` would work, though. We can add the shebang for Gleam together with a comment explaining what it's for (just like in the example you pasted above).

Reply:
I understood this comment as create a subtype of `CppSingleCommentStyle` and add the `SHEBANG` that Gleam uses. I also added comments, but I can use more guidance if this is not correct.

Thanks a lot for taking the time to review this PR.

Reply:
I see. That's close to what I meant: I meant just using `CppSingleCommentStyle` directly and expanding it with the Gleam shebang. Sorry for the confusion.

Reply:
I hope I got it right this time around :) 
Thanks again for the feedback, and sorry for the mistakes, Python is not my strong suite.